resources/js/helpers.js

```diff
@@ -391,6 +391,11 @@ export function natCol(nation, variant) {
 		// Recognise this as the color of bugs.
 		return "#ff00ff";
 	}
+	// Use Variant.NationColors if set.
+	if (variant.Properties.NationColors &&
```

Comment:
Shouldn't user defined colors override variant defined colors?

Reply:
They do, unless it's a user-defined positional colour.  The order I've set up here is:
* User defined variant-specific nation-specific colour
* User defined nation-specific colour
* Default variant-specific colour
* User defined order-based colour
* Default order-based colour

Reply:
I'm not against swapping 3 and 4 over. This was just easiest to implement